packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/__tests__/quanticGeneratedAnswer.test.js

```diff
@@ -778,6 +782,33 @@ describe('c-quantic-generated-answer', () => {
           expect(generatedAnswerCardNoAnswer).not.toBeNull();
         });
       });
+
+      describe('and when no results are returned', () => {
```

Comment:
Did you mean to write something before the `and`? or you meant just `when no results are returned`.

Please address.

Reply:
it was so that it would be more readable when running the tests:
<img width="500" alt="image" src="https://github.com/user-attachments/assets/190cbbd2-b0a9-4359-acad-8067057d8ce5" />

Reply:
it reads: `when the answer cannot be generated after a query is executed  => and when no results are returned => should not display....`

instead of :

`when the answer cannot be generated after a query is executed  =>  when results are also not returned => should not display....`

Reply:
well it's not very readable when read alone or when reading the code. So I would prefer a more explicit test description.

Reply:
I see your point, I usually read the describe blocks when I read a test to get the full context so to me it makes sense, but im fine with whatever here